aspire_cloud_hosting_policy.md

```diff
@@ -0,0 +1,53 @@
+# Aspire Cloud Hosting Policy
+
+## Preamble
+
+The goal of Aspire Cloud Repository is to provide a robust distributed source for downloading software for extending WordPress and related compatible CMSs. The Distributed Software Repository project is released freely in order to establish a network of trusted repositories which democratize software distribution free from the control of any one person or organization.
+
+This DRAFT policy governs what software will be accepted to host on AspirePress Cloud. At the project's inception, we are providing a general list of requirements and deferring to the corresponding [guidelines available at wordpress.org](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/) for interpretation until they have been fully rewritten in this document. While this document is in draft form, in case of conflicts or updates, the guideline explanations below will supersede any others.
+
+If you believe you have a case where a software package would not be accepted for hosting on wordpress.org, but believe it should be allowed on Aspire Cloud, please contact support@aspirepress.org.
+
+While this Preamble should help understand and interpret the Hosting Policy, it is not a formal part of Policy.
+
+-----
+
+## Aspire Cloud Guidelines, Terms, & Conditions
+
+### Licensing & Other Requirements
+
+1. Plugins must be compatible with the GNU General Public License v2, but may use any license which qualifies.
+2. By submitting your software and updates, you agree to indemnify and hold harmless AspirePress and its network partners for any actions resulting from your software.
+3. Code must be (mostly) human readable.
+4. Developers are responsible for the contents and actions of their plugins.
+5. A stable version of a plugin must be available from its WordPress Plugin Directory page.
+6. Plugins must respect trademarks, copyrights, and project names.
+7. Developers and their plugins must not do anything illegal, dishonest, or morally offensive.
+8. A complete plugin must be available at the time of submission.
+
+### Features & Functions Not Permitted
+
+8. Trialware is not permitted.
+9. Plugins may not track users without their explicit consent, and must offer a means for the user to opt out.
+10. Plugins may not send executable code via third-party systems.
+
+### Best Practice Guidelines
+
+11. Plugins should not hijack the admin dashboard.
+12. Plugins must use WordPress’ default libraries.
+13. Frequent commits to a plugin should be avoided.
```

Comment:
We're not using fucking subversion (the f-bomb is compulsory when talking about svn) so we don't need .org's reason for it, but I have seen packages on CPAN banned for having daily updates for no other purpose than to keep the package in the "recent changes" feed.  

Suggest replacing with "Plugins and themes MUST NOT publish version updates for the sole purpose of promoting their visibility in recent updates searches.  Patches for security and bug fixes are allowed any time, but otherwise a plugin's release cadence SHOULD be no more than weekly."